BookPlayer/Player/PlayerViewController.swift

```diff
@@ -110,17 +110,19 @@ class PlayerViewController: UIViewController, UIGestureRecognizerDelegate {
     }
 
     override func viewDidAppear(_ animated: Bool) {
-        if UserDefaults.standard.bool(forKey: Constants.UserDefaults.autolockDisabled.rawValue) {
-            UIApplication.shared.isIdleTimerDisabled = true
-        }
+        UIDevice.current.isBatteryMonitoringEnabled = true
```

Comment:
This is being enabled regardless if the setting is on or off. We should check that to avoid listening when the setting is off.